pkg/dogstatsd/server.go

```diff
@@ -782,6 +822,13 @@ func (s *Server) storeMetricStats(sample metrics.MetricSample) {
 	ms.Tags = strings.Join(s.debugTagsAccumulator.Get(), " ") // we don't want/need to share the underlying array
 	s.Debug.Stats[key] = ms
 
+	name := fmt.Sprintf("Metric Name: %s |", ms.Name)
+	tags := fmt.Sprintf(" Tags: {%s} |", ms.Tags)
+	count := fmt.Sprintf(" Count: %d |", ms.Count)
+	ls := fmt.Sprintf(" Last Seen : %d", ms.LastSeen)
+
+	s.Logger.Info(name + tags + count + ls)
```

Comment:
How frequently does this function run? Roughly how long will it take to fill up the 10mb log file?

Reply:
- Brian and I talked and this function is in a hot path so it get ran everytime a [server worker](https://github.com/DataDog/datadog-agent/blob/main/pkg/dogstatsd/server_worker.go#L50-L66) is ran.

- I took a quick test sending roughly 40k metrics with 1s pause each round. This took a whole day to filled up the 10MB. So ideally if they turn on the env var, we would have roughly 3 days worth of metrics as rollover